be2-scala/src/main/scala/ch/epfl/pop/model/network/Parsable.scala

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.pop.model.network
+
+import ch.epfl.pop.model.network.method.message.data.MessageData
+
+trait Parsable {
+  def buildFromJson(messageData: MessageData, payload: String): Any
```

Comment:
I don't understand this. What's payload? What do we expect to use this for?

A bit of documentation here might help.

Reply:
Taking a note of this! Will do if we end up keeping this